frontend/POST/methodPOST.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+import getCookie from './getCookie'
+
+const axiosInstance = axios.create({
+  headers: {"Authorization" : `Bearer ${getCookie}`,
```

Comment:
Dit is sowieso kapot, moet deze gehele file niet verwijderd worden aangezien axiosInstance.js al bestaat?